dbusmock/__init__.py

```diff
@@ -11,7 +11,7 @@
 __email__ = 'martin.pitt@ubuntu.com'
 __copyright__ = '(c) 2012 Canonical Ltd.'
 __license__ = 'LGPL 3+'
-__version__ = '0.16.3'
+__version__ = '0.16.4'
```

Comment:
This gets done automatically on release, commits shouldn't touch this. Please revert.